docs/theme/src/zh/guide/markdown/echarts.md

```diff
@@ -0,0 +1,440 @@
+---
+title: ECharts
+icon: rank
+index: 9
```

Comment:
I want to set the `ECharts` to be next to `Chart`, if you don't like this way, it may be `index: 17`?
But there has duplicate `index: 16`(external.md and others.md).
`index: 18` may be correct.

---------

英语不好，感觉想从中间插入时，所有的文件的 `index` 都要变动。
个人倾向于 `index` 设置冗余位，即使从中间插入时，不用每个文件重新修改。
比如，`index` 的步长为10，从中间插入时，有9个空位可用：

intro: `index: 10`
container: `index: 20`
code-group: `index: 30`

intro 和 container 中间插入时，可以设为 `index: 11~19`

个人意见，仅供参考。

Reply:
Yes, structure should be used in places where the sequence is not sensitive.

I just want to make an example here so I use structure. And I know it's not working good here.

So you should modify all the files behind it to plus one, and sorry for the inconvenience

Reply:
Got it, I will modify the files.